src/components/tabs/tab/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { useTheme, useMode } from 'utils/hooks';
+import { Icon } from 'components';
+import { useCombinedRefs } from 'utils/hooks/combinedrefs';
+import { useTabContext, getTabPanelId } from '../tabContext';
+import { StyledTab } from './style';
+
+export interface TabProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  index: number;
+  className?: string;
+  iconClassName?: string;
+  icon: HTMLObjectElement;
```

Comment:
icon is required but you still checking it down below. I think it should be possible to use tabs without icons, so make it optional